src/storage-client/src/controller.rs

```diff
@@ -87,6 +88,16 @@ pub enum IntrospectionType {
 pub enum DataSource {
     /// Ingest data from some external source.
     Ingestion(IngestionDescription),
+    /// This source receives its data from the identified ingestion,
+    /// specifically the output identified by `external_reference`.
+    IngestionExport {
+        ingestion_id: GlobalId,
+        // This is an `UnresolvedItemName` because the structure works for PG,
+        // MySQL, and load generator sources. However, in the future, it should
+        // be sufficiently genericized to support all multi-output sources we
+        // support.
```

Comment:
I don't think this is an actual `TODO` because we have no plan at the moment to actually do anything with this...ever AFAIK. The comment is just a very preemptive comment to a future engineer going, "Why does this need to be this way."